Makefile

```diff
@@ -145,6 +145,18 @@ FC_VER := $(FC_VER_MAJOR).$(FC_VER_MINOR).$(FC_VER_PATCH)
 LD_FLAGS        :=
 EXTRA_LD_FLAGS  :=
 
+#
+# Setup locale
+#
+LOCALE_LIST := en da
```

Comment:
This can be maintained by the directory structure :)

```
LOCALE_LIST := $(sort $(notdir $(patsubst %/,%,$(dir $(wildcard $(ROOT)/src/locales/*/locale.h)))))
```

Reply:
Very smart way, indeed changed :-)